### ISSUE TITLE
Forms - Populate lines in related tables 

### ISSUE THREAD

**Issue by pajachiet** [gristlabs/grist-core#956]
Hi,

I like the fact that Grist forms are directly integrated with the underlying data structure, especially that Reference field values can be populated with data from the referenced table.

I would love that one could populate lines in related tables. 

For example, I have a grist document to manage projects. 
When I add a new Project with a form on the Projects table, I would like to have the option to fill multiples lines of the related Roles and Deliverable tables. 

This would need **nested forms**, with multiple possible entries for the same entry in the table from the "main" form.

When only a few columns are filled with the nested form, a specific building block could be created, similar to "Columns" , but with a table.

**Comment by QurQuma** [comment]
this would be helpful for me too!
and makes sense for relationship between tables

**Comment by fisher-j** [comment]
YES this. I currently use ArcGIS Survey123, which is based on [XLSForms](xlsform.org) (which in-turn, is based on ODK XForm standard). The standard supports the idea of "repeats" which is adding records to related tables. I consider this the killer feature of this platform, and find indispensable for a wide range of data collection activities. I would love to see this capability implemented in Grist. It's rare for me to be authoring a form where I'm *not* wanting to submit form entries across relational structures.
